tests/fsharp/Libraries/ControlTests.fs

```diff
@@ -0,0 +1,176 @@
+﻿namespace FSharp.Libraries.UnitTests
+
+open System
+open NUnit.Framework
+open FSharp.Compiler.UnitTests
+
+[<TestFixture>]
+module ControlTests =
+    // Regression for FSHARP1.0:5969
+    // Async.StartChild: error when wait async is executed more than once
+    [<Test>]
+    let ExecuteAsyncMultipleTimes() =
+        CompilerAssert.CompileExeAndRun
+            """
+module M
+
+let a = async {
+                let! a = Async.StartChild(
+                            async {
+                                do! Async.Sleep(500)
```

Comment:
Since we're looking at it, do we think a full 2 seconds is necessary for this test? cc @dsyme 

If we could make this shorter that'd be great; generally I also want to make sure we're shortening the time it takes for tests to execute overall

Reply:
True, it's probably not necessary for the test to still hold.